Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/Definitions/Result.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+
+namespace Frends.AzureBlobStorage.ListBlobsInContainer.Definitions
+{
+    /// <summary>
+    ///     Returns list of container's blobs.
+    /// </summary>
+    public class Result
+    {
+        /// <summary>
+        ///     Blob's type.
+        /// </summary>
+        public string BlobType { get; set; }
+
+        /// <summary>
+        ///     Blob's URI.
+        /// </summary>
+        public string Uri { get; set; }
+
+        /// <summary>
+        ///     Blob's name.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        ///     Blob's entity tag.
+        /// </summary>
+        public string ETag { get; set; }
+
+        /// <summary>
+        ///     List of container's blobs.
+        /// </summary>
+        public List<Result> BlobList { get; set; }
```

Comment:
You have an object which contains list of objects of itself? This does not work. You should create a separate class for one blob and then have a list of blob-objects in Result-class. Also, you should probably use array instead of list, since other tasks also use arrays instead of lists. Also, please change the setter to `private set`.

Reply:
Code modified. Blob data and Result are in their own classes now. Using List here because community task was using it either.